### PR TITLE
Add correct version dependency.

### PR DESCRIPTION
**Pull request by mparker17**
Support for pear was added in the php cookbook version 1.1.0.

### CODE REVIEW

**Comment by patcon** [comment]
Sure thing, I'm all for adding more restrictive constraints! but if we're going to do it, isn't it `>= 0.99.0` when pear resource was introduced?

**Comment by mparker17** [comment]
The earliest release of the PHP cookbook which was tagged in Git was 1.1.0 (see https://github.com/opscode-cookbooks/php/releases). I haven't confirmed how the Chef code checks version numbers, but other cookbooks appear to reference git tag names.

**Comment by patcon** [comment]
Any version from the community site api is valid, which goes back further than the tags :)

**Comment by mparker17** [comment]
@patcon I've added a commit to fix the dependency as you suggested.

File: metadata.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@
 description      "Installs drush, the Drupal Shell."
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.rdoc'))
 version          "0.10.0"
-depends          "php"
+depends          "php", "~> 0.99.0"
```
Comment (patcon) [inline metadata.rb:8]: I think the cautious operator would break things, since it would cap the acceptable version at `0.99.x`, right?

**Comment by mparker17** [comment]
Oh, wow, I didn't realize that was a caution operator.

Still, though, everything seemed to work fine in my runlists... I didn't get any problems.

Thanks for your work on this!

**Pull request closed without merging**